bundle/internal/schema/annotations.yml

```diff
@@ -452,6 +452,16 @@ github.com/databricks/cli/bundle/config/resources.Permission:
   "user_name":
     "description": |-
       The name of the user that has the permission set in level.
+github.com/databricks/cli/bundle/config/resources.Pipeline:
+  "allow_duplicate_names":
+    "description": |-
+      PLACEHOLDER
+  "dry_run":
+    "description": |-
+      PLACEHOLDER
+  "run_as":
```

Comment:
Both `run_as` and `dry_run` don't seem to be supported in terraform? For `run_as` we should add support, but for `dry_run` we can either add support or remove it from the `bundle schema`.

Note, we can also switch over to using `CreatePipeline` in TF as well.

Reply:
run_as will be supported in the next TF release, the rest indeed is not

Reply:
That would be great, then at least they won't be discoverable through autocompletion.